### PR TITLE
Release 2.3.0

### PR DESCRIPTION
**Pull request by benmosher**
anything obviously necessary pre-publish? I'd like to just get it pushed if there are no red flags.

### CODE REVIEW

**Comment by ljharb** [comment]
I'm confused; what's in this branch that isn't already in master? I'm not familiar with this flow for releasing.

File: resolvers/webpack/CHANGELOG.md [hunk 1]
```diff
@@ -6,6 +6,19 @@ This change log adheres to standards from [Keep a CHANGELOG](http://keepachangel
 ## Unreleased
 
 
+## 0.8.1 - 2017-01-19
+### Changed
+- official support for Webpack 2.2.0 (RC), thanks [@graingert]
```
Comment (duncanbeevers) [inline resolvers/webpack/CHANGELOG.md]: 👏

**Comment by duncanbeevers** [comment]
I gave this a basic look-over and didn't see any show-stoppers. The boat has sailed on the constituent PRs so I'm not commenting on implementation.

The Readme and Changelog changes appear to be complete.

**Comment by benmosher** [comment]
@ljharb: nothing. I just do this to get a final confirmation CI is working + it highlights the set of commits since the last release. 

I'll merge and publish when I get to my desk.

**Comment by benmosher** [comment]
published!

**Pull request merged**